test/FileService.Mappers.UnitTests/Properties/Resources.resx

```diff
@@ -0,0 +1,128 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
+    Microsoft ResX Schema 
+    
+    Version 2.0
+    
+    The primary goals of this format is to allow a simple XML format 
+    that is mostly human readable. The generation and parsing of the 
+    various data types are done through the TypeConverter classes 
+    associated with the data types.
```

Comment:
What is this file for?

Reply:
for tests I need a big image, it is 60k symbols. I add resources for the mapper. When I added resources this 2 files were added automatically

Reply:
private string _content = Properties.Resources.SmallImageContent;
private string _bigContent = Properties.Resources.BigImageContent;
Now it looks like that, If I won't use resources It will be two strings with 6k and 60k symbols. I guess, It's unread